ui/src/pages/Tenant/helper.ts

```diff
@@ -179,6 +179,12 @@ export function checkIsSame(
   return true;
 }
 
+/**
+ * The minimum value of available resources in the currently selected zone 
+ * is the maximum value that can be entered in the input box.
+ * 
+ * The reason is to ensure that the input resource specifications can be created by each zone
+ */
 export function findMinParameter(
   zones: string[],
```

Comment:
Is it possible that `zones` is empty here? If so, the result would be a empty object and accessing its fields like `result.maxCPU` will cause problem.